sys/include/net/cord/config.h

```diff
@@ -26,41 +26,56 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_cord_conf CoRE RD endpoint and client compile configurations
+ * @ingroup config
+ * @{
+ */
 /**
  * @brief   Default lifetime in seconds (the default is 1 day)
  */
-#ifndef CORD_LT
-#define CORD_LT                 (86400UL)
+#ifndef CONFIG_CORD_LT
+#define CONFIG_CORD_LT                 (86400UL)
 #endif
 
 /**
- * @brief   Delay until the RD client starts to try registering (in seconds)
+ * @brief   Default client update interval (default is 3/4 the lifetime)
  */
-#ifndef CORD_STARTUP_DELAY
-#define CORD_STARTUP_DELAY      (3U)
+#ifndef CONFIG_CORD_UPDATE_INTERVAL
+#define CONFIG_CORD_UPDATE_INTERVAL    ((CONFIG_CORD_LT / 4) * 3)
 #endif
+/** @} */
 
 /**
- * @brief   Default client update interval (default is 3/4 the lifetime)
+ * @brief   Delay until the RD client starts to try registering (in seconds)
  */
-#ifndef CORD_UPDATE_INTERVAL
-#define CORD_UPDATE_INTERVAL    ((CORD_LT / 4) * 3)
+#ifndef CORD_STARTUP_DELAY
+#define CORD_STARTUP_DELAY      (3U)
 #endif
 
 /**
- * @name    Endpoint ID definition
+ * @brief    Endpoint ID definition
  *
  * Per default, the endpoint ID (ep) is generated by concatenation of a user
- * defined prefix (CORD_EP_PREFIX) and a locally unique ID (luid) encoded in
+ * defined prefix @ref CORD_EP_PREFIX and a locally unique ID (luid) encoded in
  * hexadecimal formatting with the given length of characters
- * (CORD_EP_SUFFIX_LEN).
+ * @ref CORD_EP_SUFFIX_LEN.
  *
  * Alternatively, the endpoint ID value can be defined at compile time by
- * assigning a string value to the CORD_ED macro.
+ * assigning a string value to the @ref CONFIG_CORD_EP macro.
```

Comment:
I don't think `@ref` is needed here as there is actually no definition for the macro.